homeassistant/components/nest/config_flow.py

```diff
@@ -239,25 +273,108 @@ async def async_step_reauth_confirm(
             entry = next(iter(existing_entries))
             if "auth_implementation" in entry.data:
                 data = {"implementation": entry.data["auth_implementation"]}
-                return await self.async_step_user(data)
+                return await super().async_step_user(data)
```

Comment:
I'm looking at this now and thinking maybe this isn't the best way to structure.
Maybe instead this branch of checking the auth impl needs to happen in self.user with the dispatch up to super user?

Reply:
I don't have an opinion at the moment about this. I think you know best.